util/leaktest/add-leaktest.sh

```diff
@@ -10,7 +10,7 @@
 # Usage: add-leaktest.sh pkg/*_test.go
 
 for i in "$@"; do
-    sed -i '' -e '
+    sed -i'' -e '
```

Comment:
With the space removed, the empty quotes are meaningless (removed by the shell). Unfortunately it looks like this is an incompatibility between GNU sed on linux and BSD sed on OSX (http://stackoverflow.com/questions/5694228/sed-in-place-flag-that-works-both-on-mac-bsd-and-linux). Make it `-i'~'` to have it leave backup files instead (we're already gitignoring `*~`).